### ISSUE TITLE
Test Google Analytics with BrowserupProxy/MITMProxy

### ISSUE THREAD

**Issue by nicholaswkc34** [selenide/selenide#2148]
## The problem
I want to test the feature of google analytics like page view, bounce rate, Average session duration. How to achieve it with Selenide?

Any articles or documentation mention it will be good. 

Example of Google Analytics feature test: 
https://www.techtarget.com/searchbusinessanalytics/definition/Google-Analytics

## Details
I want to test the feature of google analytics like page view, bounce rate, Average session duration. How to achieve it with Selenide?

Example of Google Analytics feature test: 
https://www.techtarget.com/searchbusinessanalytics/definition/Google-Analytics

## Tell us about your environment
* **Selenide Version**:  6.8.1
* **Chrome\Firefox\IE Version**: 110.0
* **Browser Driver Version**: 
* **Selenium Version**: 4
* **OS Version**: Win 10

**Comment by nicholaswkc34** [comment]
Please help me on this problem as i need this for my company project. A billion thanks for your help.

**Comment by asolntsev** [comment]
@nicholaswkc34 I am afraid I don't understand what exactly do you need. :(

**Comment by nicholaswkc34** [comment]
OK, let me rephrase my questions: 

Testing bounce rate, how many visitor for the past days of google analytics in a webpage. 
https://www.techtarget.com/searchbusinessanalytics/definition/Google-Analytics

**Comment by asolntsev** [comment]
Sorry, I still don't understand.
Anyway, this question is not related to Selenide, right? It's rather a generic test automation topic.
